Source/WebKit/UIProcess/Inspector/win/InspectorResourceURLSchemeHandler.cpp

```diff
@@ -35,24 +36,25 @@
 
 #if USE(CURL)
 #include <curl/curl.h>
+#else
+#error Unknown network backend
 #endif
 
 namespace WebKit {
 
 void InspectorResourceURLSchemeHandler::platformStartTask(WebPageProxy&, WebURLSchemeTask& task)
 {
-#if USE(CF) && USE(CURL)
     auto requestURL = task.request().url();
-    auto requestPath = requestURL.fileSystemPath();
-    if (requestPath.startsWith("\\"_s))
+    auto requestPath = requestURL.path();
+    if (requestPath.startsWith("/"_s))
         requestPath = requestPath.substring(1);
-    auto path = URL(adoptCF(CFBundleCopyBundleURL(WebCore::webKitBundle())).get()).fileSystemPath();
-    path = FileSystem::pathByAppendingComponent(path, "WebInspectorUI"_s);
-    path = FileSystem::pathByAppendingComponent(path, requestPath);
+    auto path = InspectorResource::uiPath(requestPath);
     bool success;
     FileSystem::MappedFileData file(path, FileSystem::MappedFileMode::Private, success);
     if (!success) {
+#if USE(CURL)
```

Comment:
Why this conditional? The file already does `#error` above `if !USE(CURL)`. Is this file intended to compile in non-`USE(CURL)` cases, like in the Apple Windows platform?

Reply:
A hypothetical Apple Win COULD use this file if so desired. The part thats curl specific is the error being reported.

Reply:
We don’t need #if statements for hypothetical ports. I suggest leaving the conditional out.